apps/src/music/player/sound.js

```diff
@@ -0,0 +1,118 @@
+import WebAudio from './soundSub';
```

Comment:
One other q -- is all of this new relative to our existing Sound.js file? Or could be refactored/combined at some point? Or do we intentionally want to keep this separate?

Reply:
This is all new.  I looked at the existing code but found it a bit confusing, especially since it supported both Web Audio and HTML Audio in the one file.  (A great candidate for a factory that provides one or the other implementation, as it happens.). I had this code from previous hobby projects and it fit the needs well here.  Certainly open to a unification, and would love if we had a common library that served all needs, but that feels like a nontrivial effort by itself.

Reply:
I also find the existing code supporting both audio types in one file pretty confusing, wonder if we could split it out at some point.